api/Element.json

```
@@ -7427,13 +7427,16 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/scrollLeft",
           "support": {
             "chrome": {
-              "version_added": "43"
+              "version_added": "43",
+              "notes": "Uses 0-100 for RTL instead of negative values. See <a href='https://crbug.com/721759'>issue 721759</a>"
```

* putting the context for this note first
* issue → bug, house style

```suggestion
              "notes": "For right-to-left elements, this property uses 0-100 (most left to most right) instead of negative values. See <a href='https://crbug.com/721759'>bug 721759</a>."
```